pkgs/applications/misc/kitty/default.nix

```diff
@@ -67,18 +67,6 @@ buildPythonApplication rec {
       libstartup_notification = "${libstartup_notification}/lib/libstartup-notification-1.so";
     })
   ] ++ stdenv.lib.optionals stdenv.isDarwin [
-    (fetchpatch {
-      name = "macos-symlink-1";
-      url = https://github.com/kovidgoyal/kitty/commit/bdeec612667f6976109247fe1750b10dda9c24c0.patch;
-      sha256 = "1d18x260w059qag80kgb2cgi2h4rricvqhwpbrw79s8yxzs7jhxk";
-    })
-    (fetchpatch {
-      # fixup of previous patch
-      name = "macos-symlink-2";
-      url = https://github.com/kovidgoyal/kitty/commit/af2c9a49b1ad31e94242295d88598591623fbf11.patch;
-      sha256 = "0k3dmgbvmh66j8k3h8dw6la6ma6f20fng6jjypy982kxvracsnl5";
-    })
-    ./macos-10.11.patch
```

Comment:
this change was merged in https://github.com/kovidgoyal/kitty/pull/1430

Reply:
This change was not merged. The pull request was rejected because kovidgoyal is not interested in providing support for more than 2-3 macOS versions. But I don't think anyone complained about this being broken on macOS 10.11 (I could be wrong though).

Reply:
Yes, I thought this was strange too. Anyway, we did an upgrade to 10.12 SDK, so the patch was to be removed.